transformation/common/src/main/java/org/jboss/tools/fuse/transformation/model/xml/XmlModelGenerator.java

```diff
@@ -214,6 +216,12 @@ public JCodeModel generateFromInstance(File instanceFile,
                                            File schemaFile,
                                            String packageName,
                                            File targetPath) throws Exception {
+    	// Step 0 - avoid issue with schema loading on Java 8
+    	if (System.getProperty("java.xml.accessExternalSchema") != null) {
+    		_accessExternalSchemaValue = System.getProperty("java.xml.accessExternalSchema");
+    	}
+        System.setProperty("javax.xml.accessExternalSchema", "all");        
```

Comment:
Is it this property? https://docs.oracle.com/javase/7/docs/api/javax/xml/XMLConstants.html#ACCESS_EXTERNAL_SCHEMA
Shouldn't we use the constant?